src/components/panel/panel.js

```diff
@@ -1989,6 +2049,20 @@ MdPanelPosition.prototype._setPanelPosition = function(panelEl) {
   }
 };
 
+MdPanelPosition.prototype._reverseXPosition = function (position) {
+  if (position === MdPanelPosition.xPosition.CENTER) {
+    return;
+  }
+
+  var start = 'start', end = 'end';
+
+  return position.indexOf(start) > -1 ? position.replace(start, end) : position.replace(end, start);
+};
+
+MdPanelPosition.prototype._bidi = function (position) {
```

Comment:
nit: Missing jsdoc, two lines between methods, and no space between function and params.